src/components/common/Input/Input.js

```diff
@@ -0,0 +1,39 @@
+import propTypes from "prop-types";
+import React from "react";
+import styles from "./Input.module.css";
+import classNames from "classnames";
+class Input extends React.Component {
```

Comment:
Convert this component to a functional one. See here an example https://egghead.io/lessons/react-migrate-a-react-class-component-to-a-function-component